.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,9 @@
+- id: trivy
+  name: trivy
+  entry: trivy
+  language: golang
```

Comment:
Do we need this hook? I think users will only use `trivyfs` hook. What do you think?
P.S. I tried to use this hook but can't set `PATH` to `args`. I got  ` FATAL   multiple targets cannot be specified`.

Reply:
Yeah, I also misread something in the pre-commit docs, the reason why it works for the `trivyfs` hook but not the `trivy` one is that I configured `trivyfs` with [pass_filenames](https://pre-commit.com/#hooks-pass_filenames) `: false`

pre-commit per default will not check all the files but only the staged one, therefore I think you got the error `FATAL multiple targets cannot be specified` because pre-commit executed trivy like this:
`trivy fs . staged-file1.txt  staged-file2.txt`


I'll remove the "plain hook" for now as it makes only sense if you can execute a command allows the appending of multiple files to check